projects/demo/src/modules/components/input-password/examples/2/index.ts

```diff
@@ -0,0 +1,41 @@
+import {Component} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {
+    TUI_INPUT_PASSWORD_DEFAULT_OPTIONS,
+    TUI_INPUT_PASSWORD_OPTIONS,
+} from '@taiga-ui/kit/components/input-password/input-password-options';
+import {changeDetection} from '../../../../../change-detection-strategy';
+import {encapsulation} from '../../../../../view-encapsulation';
+
+export function hideIcon(): string {
+    return 'tuiIconEyeClosed';
+}
+
+export function showIcon(): string {
+    return 'tuiIconEyeOpen';
+}
+
+@Component({
+    selector: 'tui-input-password-example-2',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+    providers: [
+        {
+            provide: TUI_INPUT_PASSWORD_OPTIONS,
+            useValue: {
+                ...TUI_INPUT_PASSWORD_DEFAULT_OPTIONS,
+                hintMode: 'onDark',
+                icons: {
+                    hide: hideIcon,
```

Comment:
This can be just plain `string`. It should be `PolymorpheusContent` which is basically a union of `string` | `(context: C) => string` | `TemplateRef<C>` | `PolymorpheusComponent<C>`. See my comment in token definition,

Reply:
Yep, I agree